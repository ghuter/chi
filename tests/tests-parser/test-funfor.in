hello :: fun () {
	for i := 0; i < 10; i += 1 {
		puts("Hello world!");
	}
}
