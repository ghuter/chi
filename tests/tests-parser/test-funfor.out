SFUN(hello, {SFOR(SDECL(i, ECSTI(0)) : ?, EBINOP(LT, EMEM(i), ECSTI(10)) : ?, SASSIGN(i, EBINOP(ADD, EMEM(i), ECSTI(1)) : ?), SCALL(puts, ECSTS(Hello world!)))}) : void
